site/en/tutorials/load_data/numpy.ipynb

```diff
@@ -120,6 +120,7 @@
         " \n",
         "import numpy as np\n",
         "import tensorflow as tf"
```

Comment:
Needs a "," at the end of line---causing a JSON formatting error

Reply:
```suggestion
        "import tensorflow as tf",
```